src/main/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStoreConfig.java

```diff
@@ -356,6 +359,13 @@ public FormValidation doCheckCustomEndpoint(@QueryParameter String customEndpoin
         return ret;
     }
 
+    public FormValidation doCheckUseHttp(@QueryParameter boolean useHttp) {
```

Comment:
```suggestion
    public FormValidation doCheckUseHttp(@QueryParameter boolean useHttp) {
        Jenkins.get().checkPermission(Jenkins.ADMINISTER);
```
otherwise we are leaking the FIPS status to anyone with Overall/Read